tests/fiaas_deploy_daemon/test_e2e.py

```diff
@@ -86,8 +102,27 @@ def wrapped():
 
         return wrapped
 
-    @pytest.fixture(autouse=True)
+    @pytest.fixture
+    def kind_logger_per_app_service_account(self, kubernetes_per_app_service_account):
+        @contextlib.contextmanager
+        def wrapped():
+            start_time = datetime.now()
+            try:
+                yield
+            finally:
+                kubernetes_per_app_service_account["log_dumper"](since=start_time, until=datetime.now())
+
+        return wrapped
```

Comment:
It might be possible to extract a function that takes "kubernetes" as a parameter and defines and returns wrapped, to avoid repetition kind_logger and in kind_logger_per_app_service_account

Reply:
good point, i missed this one